### DISCUSSION TITLE
Current Partition

### DISCUSSION THREAD

**Discussion by feder-ico** [taligentx/dscKeybusInterface#117] [dscKeybusInterface Discussion]
Hi, 
Is there any way to know witch is the current "active" partition?

I mean, if I send `/` and then `2` I switch to the partition 2 and if I send `w` will arm_away that partition.
Now I go for a couple of drinks and when I come back I don´t remember in wich partition I left it. I want to enter a code for disarming partition 1 (but the system is actually on Partition 2 where I left it last time) and when I enter the code I am really disarming the partition 2 intead of the desired 1

I there any variable that holds the last partition or a way to query the panel? 

Thanks a lot and best regads.

**Comment by taligentx** [comment]
Yes, the write partition number 1-8 is stored in `byte dsc.writePartition` - you should be able to retrieve the value in the sketch.

**Comment by feder-ico** [comment]
Thanks!
